internals/secrethub/rm_test.go

```diff
@@ -0,0 +1,270 @@
+package secrethub
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	"github.com/secrethub/secrethub-go/internals/assert"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/ui/fakeui"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/fakeclient"
+)
+
+func TestRmCommand_Run(t *testing.T) {
+
+	var testErr = errors.New("test")
+
+	cases := map[string]struct {
+		cmd                RmCommand
+		in                 string
+		argPath            api.Path
+		promptOut          string
+		out                string
+		err                error
+		promptError        error
```

Comment:
For consistency, all errors should either be named `err` or have the `Err` suffix.

Reply:
This should now be solved! :)